### ISSUE TITLE
NTR: Sulfatides in urine high (Lab: urine tests)

### ISSUE THREAD

**Issue by pnrobinson** [obophenotype/human-phenotype-ontology#8816]
New term request Sulfatides in urine high (segal_060531141527): Sulfatiduria

**Comment by pnrobinson** [comment]
@MickeySegal
Adding new term: 
- ID: HP:6000120
- label: Elevated urinary sulfatide level
- parents: [Abnormal urine metabolite level (HP:0033354)](https://hpo.jax.org/app/browse/term/HP:0033354)
- pmid: 
- def: The amount of sulfatides in the urine, normalized for urine concentration, is above the upper limit of normal. Sulfatides are derived from GalCer via esterification of a sulfate group to 3-hydroxyl of the galactose moiety.
- comment: 
- orcid: orcid:0000-0002-0736-9199
- github ID: https://github.com/obophenotype/human-phenotype-ontology/issues/8816
- synonyms:
